### PR TITLE
1486: Fixing Unit and Personnel Export to CSV and Localizing Top Menu Bar 

### LINKED ISSUE #1486: Export Personnel to CSV does not work 0.47.4
### Environment
MHQ 0.47.4
Windows 10
JAVA 1.8.0_241

### Description
Export personnel to CSV does not work. The Save File dialog box comes up. You're able to choose the folder to save the file. When you click on Save the dialog box closes and a blank information window opens. No file is generated.

### PR DESCRIPTION
**Pull request by Windchild292**
This fixes #1486, and includes a major refactor in CampaignGUI that expands the localization and orders it in the same order as the top menu bar in the campaign.

### CODE REVIEW

**Comment by Windchild292** [comment]
Oof, I'll fix those in a few minutes.

**Comment by Windchild292** [comment]
Fixed, this is now ready for review

File: MekHQ/src/mekhq/gui/CampaignGUI.java [hunk 1]
```diff
@@ -590,619 +576,386 @@ public void removeTab(String tabName) {
...
+        miImportPerson.addActionListener(this::miImportPersonActionPerformed);
+        menuImport.add(miImportPerson);
+
+        JMenuItem miImportParts = new JMenuItem(resourceMap.getString("miImportParts.text")); // NOI18N
+        miImportParts.addActionListener(this::miImportPartsActionPerformed);
+        menuImport.add(miImportParts);
+
+        JMenuItem miLoadForces = new JMenuItem(resourceMap.getString("miLoadForces.text")); // NOI18N
+        miLoadForces.addActionListener(this::miLoadForcesActionPerformed);
+        // miLoadForces.setEnabled(false);
```
Comment (sixlettervariables) [inline MekHQ/src/mekhq/gui/CampaignGUI.java]: Is this still needed?

Comment (Windchild292) [reply MekHQ/src/mekhq/gui/CampaignGUI.java]: Nope, give me 5

Comment (Windchild292) [reply MekHQ/src/mekhq/gui/CampaignGUI.java]: Fixed and push

**Pull request merged**